<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">CuteCoin</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="label_account_name">
      <property name="text">
       <string>Please select an account</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="actionsFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTabWidget" name="tabs_account">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="tab_wallets">
          <attribute name="title">
           <string>Wallets</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QListView" name="list_wallets"/>
           </item>
           <item>
            <widget class="QListView" name="list_wallet_content"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_transactions">
          <attribute name="title">
           <string>Transactions</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Sent</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListView" name="list_transactions_sent"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Received</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListView" name="list_transactions_received"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_communities">
          <attribute name="title">
           <string>Communities</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QTabWidget" name="tabs_communities">
             <property name="currentIndex">
              <number>-1</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>498</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_account">
    <property name="title">
     <string>Account</string>
    </property>
    <widget class="QMenu" name="menu_change_account">
     <property name="title">
      <string>Change account</string>
     </property>
    </widget>
    <addaction name="action_configure_parameters"/>
    <addaction name="menu_change_account"/>
    <addaction name="action_add_account"/>
    <addaction name="separator"/>
    <addaction name="action_export"/>
    <addaction name="action_import"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_contacts">
    <property name="title">
     <string>Contacts</string>
    </property>
    <widget class="QMenu" name="menu_contacts_list">
     <property name="title">
      <string>Contacts</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="action_add_a_contact"/>
    <addaction name="menu_contacts_list"/>
   </widget>
   <widget class="QMenu" name="menu_actions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionTransfer_money"/>
   </widget>
   <addaction name="menu_account"/>
   <addaction name="menu_contacts"/>
   <addaction name="menu_actions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionManage_accounts">
   <property name="text">
    <string>Manage accounts</string>
   </property>
  </action>
  <action name="actionConfigure_trustable_nodes">
   <property name="text">
    <string>Configure trustable nodes</string>
   </property>
  </action>
  <action name="action_add_a_contact">
   <property name="text">
    <string>Add a contact</string>
   </property>
  </action>
  <action name="actionSend_a_message">
   <property name="text">
    <string>Send a message</string>
   </property>
  </action>
  <action name="actionSend_money">
   <property name="text">
    <string>Send money</string>
   </property>
  </action>
  <action name="actionRemove_contact">
   <property name="text">
    <string>Remove contact</string>
   </property>
  </action>
  <action name="actionSend_a_message_2">
   <property name="text">
    <string>Send a message</string>
   </property>
  </action>
  <action name="actionSend_money_2">
   <property name="text">
    <string>Send money</string>
   </property>
  </action>
  <action name="actionRemove_contact_2">
   <property name="text">
    <string>Remove contact</string>
   </property>
  </action>
  <action name="action_add_account">
   <property name="text">
    <string>Add account</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAccount">
   <property name="text">
    <string>Account</string>
   </property>
  </action>
  <action name="actionTransfer_money">
   <property name="text">
    <string>Transfer money</string>
   </property>
  </action>
  <action name="action_configure_parameters">
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
  <action name="action_import">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="action_export">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_add_account</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_add_account_dialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTransfer_money</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_transfer_money_dialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_add_a_contact</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_add_contact_dialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_configure_parameters</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_configure_account_dialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list_wallets</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>refresh_wallet_content(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_import</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>import_account()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_export</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>export_account()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>open_add_account_dialog()</slot>
  <slot>import_account()</slot>
  <slot>open_transfer_money_dialog()</slot>
  <slot>open_add_contact_dialog()</slot>
  <slot>open_configure_account_dialog()</slot>
  <slot>refresh_main_window()</slot>
  <slot>refresh_wallet_content(QModelIndex)</slot>
  <slot>export_account()</slot>
 </slots>
</ui>
